pkg/artifactory/resource/repository/repository.go

```diff
@@ -122,9 +122,9 @@ func mkRepoUpdate(unpack unpacker.UnpackFunc, read schema.ReadContextFunc) schem
 		projectKeyChanged := d.HasChange("project_key")
 		tflog.Debug(ctx, fmt.Sprintf("projectKeyChanged: %v", projectKeyChanged))
 		if projectKeyChanged {
-			old, new := d.GetChange("project_key")
+			old, newProject := d.GetChange("project_key")
 			oldProjectKey := old.(string)
-			newProjectKey := new.(string)
+			newProjectKey := newProject.(string)
```

Comment:
Not sure these changes are necessary

Reply:
It was shadowing build-in command and IDE was issuing a warning.